src/state/slices/txHistorySlice/selectors.ts

```diff
@@ -127,6 +132,50 @@ export const selectTxIdsByFilter = createCachedSelector(
     : 'txIdsByFilter',
 )
 
+export const selectReceivedTxsForAccountIdsByFilter = createCachedSelector(
+  selectTxIds,
+  selectTxs,
+  selectWalletTxsByAccountIdAssetId,
+  (_state, filter: { accountIds: AccountId[]; txSender: string } | undefined) => filter?.accountIds,
+  (_state, filter: { accountIds: AccountId[]; txSender: string } | undefined) => filter?.txSender,
```

Comment:
issue (blocking): absolutely fine as a draft as things may change here, but we will want to use `selectXyzParamFromFilter` or this may be reactive on the whole redux store

Reply:
yikes nice catch

Reply:
fixed 775d83e5499be55bbbb3404ed418489686430fc3